src/engine/renderer/glsl_source/contrast_fp.glsl

```diff
@@ -42,15 +42,14 @@ vec4 f(vec4 color) {
 
 void	main()
 {
-	vec2 scale = r_FBufScale;
-	vec2 st = gl_FragCoord.st;
-
 	// calculate the screen texcoord in the 0.0 to 1.0 range
-	st *= r_FBufScale;
+	vec2 st = gl_FragCoord.st / r_FBufSize;
 
 	// multiply with 4 because the FBO is only 1/4th of the screen resolution
 	st *= vec2(4.0, 4.0);
 
+	vec2 scale = 1 / r_FBufSize;
```

Comment:
unused?

Reply:
It is used from line `54` and beyond. Don't worry I did the mistake first, my eyes first failed to notice them until the compiler yelled at me because I had deleted it in my first attempt.